src/dmd/globals.d

```diff
@@ -323,9 +344,10 @@ extern (C++) struct Global
         return anyErrs;
     }
 
-    /*  Increment the error count to record that an error
-     *  has occurred in the current context. An error message
-     *  may or may not have been printed.
+    /**
+     * Increment the error count to record that an error has occurred in the current context.
```

Comment:
Which context?

Reply:
I'm not sure actually, I didn't write the original comment. It's only used once for expression semantic of a `StructLiteralExp` when filling the remaining fields fails.